cy/mv.py

```
@@ -0,0 +1,147 @@
+        return
+    # ---
+    if new_text == text:
+        printe.output("no changes")
+        return
+    # ---
+    if new_text:
+        page.save(newtext=new_text, summary="بوت:تجربة تحديث بيانات اللاعب")
+
+
+def main2(*args):
+    generator = gent.get_gent(listonly=True, *args)
```

Ruff

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)